tests/TestCase/ORM/MarshallerTest.php

```diff
@@ -2586,9 +2630,9 @@ public function testAssociatoinsFieldList()
         ];
         $marshall = new Marshaller($this->articles);
         $result = $marshall->one($data, [
-            'fieldList' => ['title', 'body', 'user'],
+            $fields => ['title', 'body', 'user'],
             'associated' => [
-                'Users' => ['fieldList' => ['username', 'foo']]
+                'Users' => [$fields => ['username', 'foo']]
```

Comment:
Could we keep the old and new test alongside and mark the old tests one deprecated (with version)?